powermon/__init__.py

```diff
@@ -166,17 +154,29 @@ def main():
 
     # Main working loop
     keep_looping = True
+    controller_looping = True
+    device_looping = True
     try:
         while keep_looping:
             # tell the daemon we're still working
             daemon.watchdog()
 
             # run schedule loop
-            keep_looping = controller.runLoop() or device.runLoop()
+            if controller_looping:
+                controller_looping = controller.runLoop()
```

Comment:
If you don't like my schedule loop you can remove it. Now it's just broken because it's not properly calling the protocols properly.

Reply:
has been removed